tendrl/gluster_integration/objects/definition/gluster.yaml

```diff
@@ -60,6 +60,17 @@ namespace.gluster:
       type: Create
       uuid: b8dd2b4a-96de-4ded-b17d-374d26ec7593
       version: 1
+    SyncAggregatedClusterDetails:
+      tags:
+        - "provisioner/gluster"
```

Comment:
For this we would need to add `provisioner/{int-id}` flag while import flow for the first node which picks the import job. Will update the commons PR https://github.com/Tendrl/commons/pull/692 accordingly